challenge_01/script.js

```diff
@@ -9,55 +9,98 @@ formElements.radioBtnSR = document.querySelector("#radio-btn-SR");
 formElements.messageTextarea = document.querySelector("#message-textarea");
 formElements.consentCheckBox = document.querySelector("#consent-checkbox");
 
+const toast = document.querySelector("#toast");
+const toastText = document.querySelector("#toast-text");
+const toastExit = document.querySelector("#toast-exit");
+
+toastExit.addEventListener('click',() => {
+    toast.classList.add("hidden");
+})
+
 formElements.contactForm.addEventListener('submit',(event) => {
 
     // disabling the default submit and refresh
     event.preventDefault();
 
 
-    let shouldReturnEarly = false;
     //checking for wrong inputs
     if (!formElements.firstNameInput.value){
-        formErrorHandler("please enter first name");
+        formErrorHandler("please enter first name",formElements.firstNameInput);
+        return;
```

Comment:
You have repeated `return` 9 times. What happens if the requirements change? You have to edit 9 places in your code and this is the root of most bugs.
You can take a boolean flag and while calling `formErrorHandler`, you can flip that boolean if any error occurs.

Reply:
My bad.
I messed up in one part of code and tried to get back and got wrong version restored which still had the `return`s .